src/components/DeleteModal.js

```diff
@@ -50,7 +50,7 @@ const DeleteModal = ({ handleModalToggle, isModalOpen, currentSytem, onConfirm }
         </Split>
         <Level hasGutter>
             <LevelItem>
-                <Button variant="danger" onClick={onConfirm}>
+                <Button variant="danger" onClick={onConfirm} ouiaId="Remove">
```

Comment:
I'd change the ouiaId to be more specific `Remove` to `confirm-delete` as that's what this button does.